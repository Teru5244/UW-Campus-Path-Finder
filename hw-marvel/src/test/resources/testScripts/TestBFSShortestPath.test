# Test if the BFS is finding the shortest path between two nodes

LoadGraph simple simpleGraph.csv

FindPath simple n1 n3

FindPath simple n2 n3

FindPath simple n1 n2