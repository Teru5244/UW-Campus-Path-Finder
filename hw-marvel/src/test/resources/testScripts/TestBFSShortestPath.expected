# Test if the BFS is finding the shortest path between two nodes

loaded graph simple

path from n1 to n3:
n1 to n3 via e3

path from n2 to n3:
n2 to n3 via e2

path from n1 to n2:
n1 to n2 via e1